cuML/test/dbscan_test.cu

```diff
@@ -69,10 +70,14 @@ protected:
 		int min_pts = 2;
 		cumlHandle handle;
 		handle.setStream(stream);
-		dbscanFit(handle, data, params.n_row, params.n_col, eps, min_pts, labels);
+
+		// forces a batch size of 2
```

Comment:
Sorry for my basic question, but why does max_elems=10 force a batch size of 2? 

Also, cast to (size_t) not necessary